HGCalAnalysis/plugins/HGCalAnalysis.cc

```diff
@@ -836,7 +921,75 @@ HGCalAnalysis::analyze(const edm::Event& iEvent, const edm::EventSetup& iSetup)
 		multiclus_slopeY.push_back(multiClusters[i].y());
 		multiclus_cluster2d.push_back(cl2dIndices);
 		multiclus_cl2dSeed.push_back(cl2dSeed);
+		multiclus_firstLay.push_back(*layers.begin());
+		multiclus_lastLay.push_back(*layers.rbegin());
+		multiclus_NLay.push_back(layers.size());
+		if(multiClusters[i].size()<3)
+		  	{
+			multiclus_pcaAxisX.push_back(-2.);
```

Comment:
I reran this branch on top of @clelange latest PR (https://github.com/cms-sw/cmssw/pull/19149) and now there seem to be no multiclusters with less than 2dclusters or layers. Therefore, this default value could be omitted.
Alternatively, I would suggest to just skip, i.e. not store, multiclusters with less then 3 layers.